docs/data/data-grid/filtering-recipes/filtering-recipes.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Data Grid - Filtering customization recipes
+---
+
+# Data Grid - Filtering customization recipes
+
+<p class="description">Advanced filtering customization recipes.</p>
+
+## Quck filter outside of the grid
```

Comment:
Nice use case!

Reply:
Typo fixed in https://github.com/mui/mui-x/commit/ccca7c6a0b8804045fbd27e660ebfb7c68540523.